processing_code/TutorialUniqueIDRecord.csv

```diff
@@ -149,7 +149,7 @@ b152963c4883463883c3b6f0de95fd93,Access and Work with NEON Geolocation Data,
 db9715ca243944fabbe81031f2ed5cec,Advanced Spectral Signatures in R,
 884cc5915a3842c1ab6617b060797fab,Subset HDF5 Hyperspectral,
 7a3d01f3a2a84e6092774e2d21e13a16,Structural Diversity Discrete Return,
-99d22b526c1a4a88847259a4d05b46af,,
```

Comment:
@donal-at-NEON - now that we have the dev-aten & master branches how are we avoiding duplicate syncIDs?  Should we just not assign the new tutorials in dev-aten a sync ID until we're ready for the roll over?

Reply:
Actually - realized that won't work as they then can't show up on the dev site.... Continue to use the "master" file to keep track of the syncIDs?  I'm open to other ideas.

Reply:
Would it cause problems to pre-assign some syncIDs - ie, give them filler titles before there's actually any content corresponding to that title? We could block off several, and then start the dev-aten syncIDs below the block?

Reply:
All good points! I have not assigned _any_ new syncIDs to the dev-aten branch, and I am continuing to use the 'master' branch TutorialUniqueIDRecord.csv as the main list to reference and update. I will likely begin to assign syncIDs to the new tutorials on the dev-aten branch, but at that point I will update the IDRecords.csv in _both_ branches to avoid this conflict.

Reply:
Awesome!